docs/mkdocs.yml

```diff
@@ -6,6 +6,7 @@ dev_addr: 127.0.0.1:8765
 
 repo_name: 'mozilla/bedrock'
 repo_url: 'https://github.com/mozilla/bedrock'
+edit_uri: edit/main/docs/docs/
```

Comment:
two `docs/`?

Reply:
yeah. I coppied how springfield did it. `docs/` contains the makefile and mkdocs.yml and `docs/docs/` contains all of the markdown files.